mkosi/machine.py

```diff
@@ -147,6 +148,11 @@ def boot(self) -> None:
             # Then, when pexpects finds the '#' it means we're ready to interact with the process.
             self._serial = pexpect.spawnu(command=cmd[0], args=cmd[1:], logfile=LogfileAdapter(sys.stdout),
                                           timeout=240)
+
+            if self.luks_password is not None:
+                self._serial.expect("Please enter passphrase for disk")
+                self._serial.write(self.luks_password + "\n")
+
```

Comment:
Why do we have both a password file and this code to pass the password in via the terminal? Shouldn't we just use the password file and drop this?